### PR TITLE
Drupal 9 - add upgrade path

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Summary

**[Drupal 9](https://pantheon.io/docs/drupal-9)** - Gives users the steps to upgrade to an existing site to Drupal 9. [PR](#)

## Effect

The following changes are already committed:

- put the steps back to upgrade to D9 (reverted commit)
- 

## Remaining Work

The following changes still need to be completed:

- [ ] update steps with correct numbers and content
- [ ] tech edit
- [x] copy edit

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5762 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `51` | `-3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5762` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/21433/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/21433/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/drupal-9.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Test Drupal 9 with a fresh installation using either our Integrated Composer Ear
...
 | [**Build Tools**](#create-a-drupal-9-site-with-continuous-integration-via-build-tools) | • Composer automatically runs via CI <br /> • Pull request-based workflow <br /> • Supports automated testing                                              | • Requires 3rd-party CI <br /> • Terminus required <br /> • Composer updates happen outside the 1-click Pantheon Dashboard |
 
-If you're not ready to create a new site yet, you can also [check an existing site's compatibility to upgrade](#test-an-existing-drupal-site-for-drupal-9-upgrade-compatibility).
+If you're not ready to create a new site yet, you can also [check an existing site's compatibility to upgrade](#test-an-existing-drupal-site-for-drupal-9-upgrade-compatibility). Once you're ready, [test upgrade an existing pantheon drupal 8 site to drupal 9](#test-upgrade-an-existing-pantheon-drupal-8-site-to-drupal-9).
```
Comment (alexfornuto) [inline source/content/drupal-9.md:41]: Suggest "test-upgrade" or "test upgrading".

File: source/content/drupal-9.md [hunk 2]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
 
 Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
 
+## Test Upgrade an Existing Pantheon Drupal 8 Site to Drupal 9
```
Comment (alexfornuto) [inline source/content/drupal-9.md:98]: Same as L41, but lean towards the first option.

File: source/content/drupal-9.md [hunk 3]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
...
+
+Already running a Pantheon site using our [Drupal 8 upstream](https://github.com/pantheon-systems/drops-8)? Leverage the amazing power of our Multidev tooling to test Drupal 9 in a Git branch.
```
Comment (alexfornuto) [inline source/content/drupal-9.md:100]: preface "Are you".

File: source/content/drupal-9.md [hunk 4]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
...
+
+1. Clone your Drupal 8 [site’s codebase to your computer](https://pantheon.io/docs/local-development#get-the-code). You can create a new Drupal 8 site or use an existing Drupal 8 site:
```
Comment (alexfornuto) [inline source/content/drupal-9.md:102]: localize link

File: source/content/drupal-9.md [hunk 5]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
...
+  terminus multidev:create <site>.<env> d9-beta
+  ```
+
+1. Create and switch to a new testing branch in the Drupal 8 codebase:
+
+  ```bash{promptUser: user}
+  git checkout -b d9-beta
+  ```
+
+1. Pull the `drupal9-beta3--from-8.8.6--early-access-not-for-production-use` branch into your codebase:
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:120]: We could make `9.0.0` now. It's a tag, though, not a branch.

Comment (EdwardAngert) [reply source/content/drupal-9.md]: thanks, @greg-1-anderson - I [updated that section to reflect](https://github.com/pantheon-systems/documentation/pull/5762/files#diff-207a2bff4a28714731dff2b0275b0bd0R120-R123), and left the "early access" part to keep the local name obvious. Let me know if any of that should be modified

File: source/content/drupal-9.md (hunk 4 above)
Comment (EdwardAngert) [reply source/content/drupal-9.md]: ++ on the edits
all updated to reflect

File: source/content/drupal-9.md [hunk 6]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
...
+
+Are you already running a Pantheon site using our [Drupal 8 upstream](https://github.com/pantheon-systems/drops-8)? Leverage the amazing power of our Multidev tooling to test Drupal 9 in a Git branch.
+
+1. Clone your Drupal 8 [site’s codebase to your computer](/local-development#get-the-code). You can create a new Drupal 8 site or use an existing Drupal 8 site:
+
+  ```bash{promptUser: user}
+  git clone <url for site repo>
+  ```
+
+1. Use Terminus to create a [Multidev](/multidev) environment called `d9-beta` on your Drupal 8 site for testing:
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:108]: Drupal 9 is stable now. Maybe we should call out example environment "d9-preview". This emphasizes that we have not done a full launch of Drupal 9 on the platform. "Beta" implies that the docs are just old and maybe Drupal 9 is launched now.

File: source/content/drupal-9.md [hunk 7]
```diff
@@ -95,6 +95,47 @@ $node = Node::load(1);
...
+  git checkout -b d9-beta
+  ```
+
+1. Pull the `drupal9.0.0` tag into your codebase:
+
+  ```bash{promptUser: user}
+  git pull https://github.com/pantheon-systems/drops-8.git drupal9.0.0--from-8.9.0--early-access-not-for-production-use
+  ```
+
+1. Modify the `pantheon.yml` file to specify PHP 7.4 and Drush 8:
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:126]: Should we expand on the fact that the pre-installed Drush 10 does not work with Drupal 9?

Comment (EdwardAngert) [reply source/content/drupal-9.md:126]: added a note about Drush 10
should I also add that PHP 7.3 is ok?

Comment (greg-1-anderson) [reply source/content/drupal-9.md:126]: Yeah, there was talk of making Drupal 9 require PHP 7.4, but 9.0.0 still works with 7.3.

File: source/content/drupal-9.md [hunk 8]
```diff
@@ -95,6 +95,49 @@ $node = Node::load(1);
...
+  terminus multidev:create <site>.<env> d9-preview
+  ```
+
+1. Create and switch to a new testing branch in the Drupal 8 codebase:
+
+  ```bash{promptUser: user}
+  git checkout -b d9-preview
+  ```
+
+1. Pull the `drupal9.0.0` tag into your codebase:
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:120]: `drupal9.0.0` should just be `9.0.0`

File: source/content/drupal-9.md [hunk 9]
```diff
@@ -95,6 +95,49 @@ $node = Node::load(1);
...
+
+  ```bash{promptUser: user}
+  git pull https://github.com/pantheon-systems/drops-8.git drupal9.0.0--from-8.9.0--early-access-not-for-production-use
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:123]: `drupal9.0.0--from-8.9.0--early-access-not-for-production-use` should just be `9.0.0`

File: source/content/drupal-9.md [hunk 10]
```diff
@@ -95,6 +95,49 @@ $node = Node::load(1);
 
 Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
 
+## Test-upgrade an Existing Pantheon Drupal 8 Site to Drupal 9
```
Comment (greg-1-anderson) [inline source/content/drupal-9.md:98]: I just had another thought that we shouldn't call this "Test-upgrade". Using the word "upgrade" to me sounds like deploying to production. I'd rather have a softer title here, like "Try out Drupal 9 on an Exiting Pantheon site using a Multidev environment".

We can talk about "upgrade" once we have a drops-9 upstream.

Comment (EdwardAngert) [reply source/content/drupal-9.md:98]: I tried rewriting this, but the heading without `upgrade` loses the meaning
I think @alexfornuto 's suggestion of the hyphenated version should give folks the idea that this is a trial path. So I'm letting it stand as-is and moving this to copy edit

Comment (greg-1-anderson) [reply source/content/drupal-9.md:98]: "Test an Existing Pantheon Drupal 8 Site with Drupal 9"?

**Pull request merged**